test/engine-tap/lib/tester.lua

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env tarantool
+
+box.cfg{
+    memtx_max_tuple_size = 4996109;
+    vinyl_max_tuple_size = 4996109;
+    log = "tarantool.log";
+    memtx_allocator      = os.getenv("MEMTX_ALLOCATOR")
+}
+
+local tap = require('tap')
+local test = tap.test("errno")
+
+local ok, test_run = pcall(require, 'test_run')
+test_run = ok and test_run.new() or nil
+local engine = test_run ~= nil and test_run:get_cfg('engine') or 'memtx'
```

Comment:
You could also use luatest and not struggle with this.

Reply:
How? I've created tap version of engine test several months ago, and at the moment there was no much luatest. I could missed something since then, and if such could you please show me examples how we could invoke luatest with several options varied (like memtx and vinyl here)?

Reply:
I am not a point of contact how to write luatests with multiple configs. I only know I saw it somewhere in luatest code. You would need to ask luatest adepts. Or simply not care. I will approve it anyway.